SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalanceEnforcer/SpatialLoadBalanceEnforcerTest.cpp

```diff
@@ -25,8 +25,8 @@ PhysicalWorkerName ValidWorkerTwo = TEXT("ValidWorkerTwo");
 VirtualWorkerId VirtualWorkerOne = 1;
 VirtualWorkerId VirtualWorkerTwo = 2;
 
-Worker_EntityId EntityIdOne = 0;
```

Comment:
Just interested in why this is needed? Is 0 somehow now invalid?

Reply:
I don't think you can have entities with ID 0, no- but also, it was more just misleading, as if you were debugging this and looking at the entity IDs, then `1` would be `EntityIdTwo` which seemed strange

Reply:
TRUE!